components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/utils/GatewayCmdUtils.java

```diff
@@ -222,6 +222,15 @@ public static String getResourceFolderLocation() {
                 + GatewayCliConstants.GW_DIST_RESOURCES;
     }
 
+    /**
+     * Get grpc folder location inside Resources Directory.
+     *
+     * @return grpc folder location
+     */
+    public static String getGrpcFolderLocation() {
```

Comment:
Rename all Folder references to `Dir` or `Directory`. folder is a windows specific term

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/757/commits/6142fe19275c1d5c96e88b26d6d141d335db252b